panic occurred when write a wrong tx-decode parameter name

I write a wrong tx-decode parameter name, then a panic occurred.

```bash
./qx tx-decode -n tesnet 01000000010a00000000000000000000000000000000000000000000000000000000000000feffffff02010000010000005b2048000000001976a9142421972d46cedd2283d244665a05bde2aec3446f88ac00000000000000003df06963017f354d6b4237744d6356437477556376317a77757959357a594c5a4342756e33365a47684838687347616e566878337059675a75624c4e48473044022053b6db147240f8ecc5f8b710b1f5b378404c86448bc49175605fbceedd8b98ce0220592dad6968962bf93b63cf56f68bcc5f86cf7dd7c68a1b2e08bae105c3705f5a01|jq
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x152 pc=0x430c329]

goroutine 1 [running]:
github.com/Qitmeer/qng/core/address.NewPubKeyHashAddress({0xc0000a8122?, 0x5?, 0x19?}, 0x4c9f060?, 0x0?)
	/Users/blocklee/git/qng/core/address/pkhaddr.go:66 +0x29
github.com/Qitmeer/qng/engine/txscript.ExtractPkScriptAddrs({0xc0000a811f, 0x19, 0x19}, 0x0)
	/Users/blocklee/git/qng/engine/txscript/standard.go:1258 +0x477
github.com/Qitmeer/qng/common/marshal.MarshJsonVout(0xc000b18200, 0x0, 0xd11667c8e22dfd67?)
	/Users/blocklee/git/qng/common/marshal/marshal.go:160 +0x12a
github.com/Qitmeer/qng/qx.TxDecode({0x7ff7bfeff302?, 0xc0000120c0?}, {0x7ff7bfeff309?, 0x461d325?})
	/Users/blocklee/git/qng/qx/handle_tx.go:235 +0x3ed
main.main()
	/Users/blocklee/git/qng/cmd/qx/qx.go:1402 +0x7091
```

--- a/qx/handle_tx.go
+++ b/qx/handle_tx.go
@@ -150,6 +150,8 @@ func TxSign(privkeyStrs []string, rawTxStr string, network string) (string, erro
 		param = &params.PrivNetParams
 	case "mixnet":
 		param = &params.MixNetParams
+	default:
+		ErrExit(fmt.Errorf("invalid network (mainnet|testnet|privnet|mixnet)"))
 	}
 
 	if len(rawTxStr)%2 != 0 {
@@ -200,6 +202,8 @@ func TxDecode(network string, rawTxStr string) {
 		param = &params.PrivNetParams
 	case "mixnet":
 		param = &params.MixNetParams
+	default:
+		ErrExit(fmt.Errorf("invalid network (mainnet|testnet|privnet|mixnet)"))
 	}
 	strArr := strings.Split(rawTxStr, MTX_STR_SEPERATE)
 	rawTxStr = strArr[0]

